### PR TITLE
churcill added

### PR DESCRIPTION
**Pull request by cemuka**

### CODE REVIEW

File: src/recipes/churchill.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+    "name": "Churcill",
```
Comment (alfg) [inline src/recipes/churchill.json:2]: Typo?

File: src/recipes/churchill.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            "ingredient": "Salt"
+        }
+    ],
+    "directions": [
+        "Squeeze the lemon into the glass.",
+        "Poor the mineral water, be carefull with bubbles.",
+        "Add salt.",
+        "Additionaly, you may add some ice."
+    ],
+    "image": "churcill-drink.jpg",
```
Comment (alfg) [inline src/recipes/churchill.json:28]: Image filename and json filename should match. Also, typo?

**Review by alfg — CHANGES_REQUESTED** [review]
Hi, can you rename the files to match? Should be `churchill.jpg` and `churchill.json`.

**Comment by cemuka** [comment]
Thanks for pointing out, it's my bad :)

**Comment by alfg** [comment]
Thanks!

**Pull request merged**